### PR TITLE
ECL-595: Accessibility fix

### PR DESCRIPTION
**Pull request by paddynski-opencast**

### CODE REVIEW

File: app/uk/gov/hmrc/economiccrimelevyenrolment/forms/EclRegistrationDateFormProvider.scala [hunk 1]
```diff
@@ -24,11 +24,15 @@ import javax.inject.Inject
 
 class EclRegistrationDateFormProvider @Inject() extends Mappings {
 
+  def sanitise(value: String) =
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevyenrolment/forms/EclRegistrationDateFormProvider.scala:27]: Needs a better name.

File: app/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/LocalDateFormatter.scala [hunk 2]
```diff
@@ -27,6 +27,7 @@ import scala.util.{Failure, Success, Try}
 private[mappings] class LocalDateFormatter(
   invalidKey: String,
   requiredKey: String,
+  sanitise: String => String,
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/LocalDateFormatter.scala:30]: This is appropriate here, as it's a function which could accept any type of sanitise operation. In my comment above, the function you pass in though, in the form provider should be named against what it does.

File: test/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/DateMappingsSpec.scala [hunk 3]
```diff
@@ -50,14 +52,27 @@ class DateMappingsSpec
 
   val missingField: Gen[Option[String]] = Gen.option(Gen.const(""))
 
+  def sanitise(value: String): String =
```
Comment (TheDanielDoyle) [inline test/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/DateMappingsSpec.scala:55]: Same as top comment. Either **trimSpaces** or **removeSpaces**

File: app/uk/gov/hmrc/economiccrimelevyenrolment/forms/EclRegistrationDateFormProvider.scala (hunk 1 above)
Comment (paddynski-opencast) [reply app/uk/gov/hmrc/economiccrimelevyenrolment/forms/EclRegistrationDateFormProvider.scala:27]: Renamed

File: app/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/LocalDateFormatter.scala (hunk 2 above)
Comment (paddynski-opencast) [reply app/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/LocalDateFormatter.scala:30]: Agreed and corrected.

File: test/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/DateMappingsSpec.scala (hunk 3 above)
Comment (paddynski-opencast) [reply test/uk/gov/hmrc/economiccrimelevyenrolment/forms/mappings/DateMappingsSpec.scala:55]: Renamed.

**Review by TheDanielDoyle — APPROVED** [review]

**Pull request merged**